app/src/main/java/com/aws/amazonlocation/ui/main/map_style/MapStyleFragment.kt

```diff
@@ -53,10 +70,40 @@ class MapStyleFragment : BaseFragment() {
         clickListener()
     }
 
+    private fun init() {
+        setColumnCount()
+        mLayoutManagerHere.spanCount = columnCount
+        mLayoutManagerEsri.spanCount = columnCount
+        mHereAdapter?.notifyDataSetChanged()
+        mAdapter?.notifyDataSetChanged()
+    }
+
+    private fun setColumnCount() {
+        if (isLargeTablet) {
```

Comment:
Is there a way to refactor it to make number of columns calculate from widthPixels? I assume there should be some formula from math series.